cmd/admin-handlers.go

```diff
@@ -450,26 +481,73 @@ func (a adminAPIHandlers) GetConfigHandler(w http.ResponseWriter, r *http.Reques
 		return
 	}
 
-	// Take a read lock on minio/config.json. NB minio is a
-	// reserved bucket name and wouldn't conflict with normal
-	// object operations.
-	configLock := globalNSMutex.NewNSLock(minioReservedBucket, minioConfigFile)
-	if configLock.GetRLock(globalObjectTimeout) != nil {
-		writeErrorResponseJSON(w, ErrOperationTimedOut, r.URL)
+	configBytes, err := a.getConfigDotJSON()
+	if err != nil {
+		logger.LogIf(context.Background(), err)
```

Comment:
We should create a correct context here for logging.

Reply:
> We should create a correct context here for logging.

Done